docs/getting-started/webhook-run.yaml

```diff
@@ -19,7 +19,9 @@ spec:
     value: token
   - name: GitHubSecretStringKey
     value: secret
+  - name: GitHubDomain
+    value: git.corp.com
```

Comment:
im not sure we want to add this by default here - maybe we could have it commented out? e.g.

```yaml
#  If you are using github enterprise, uncomment this line <more context here>
#  - name: GitHubDomain
#    value: git.corp.com
```